instrumentation-genai/opentelemetry-instrumentation-vertexai-v2/example/main.py

```diff
@@ -0,0 +1,16 @@
+import vertexai
+from vertexai.generative_models import GenerativeModel
+
+vertexai.init(location="us-central1")
+
+model = GenerativeModel("gemini-1.5-flash-002")
```

Comment:
similar can you use an ENV? even better to basically copy the openai example word for word then adjust the library calls etc. This reduces the amount of attention needed to land your change vs having differences in setup and also library https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation-genai/opentelemetry-instrumentation-openai-v2/examples/zero-code/main.py#L9